scripts/pa11y.js

```diff
@@ -52,24 +52,49 @@ async function runPa11y() {
   const setupBrowser = browsers[0].browser;
   const page = await setupBrowser.newPage();
 
-  await page.goto('http://localhost:3000');
-  const urls = Array.from(
-    await page.evaluate(() =>
-      [...document.querySelectorAll('a')].map((element) =>
-        element.getAttribute('href'),
-      ),
-    ),
-  );
+  const iframePath = `file://${__dirname}/../build/storybook/static/iframe.html`;
+
+  // window.__storybook_stories__ is injected into the iframe by the percy addon
+  // so that Percy's script knows what stories exist. Piggybacking off that is
+  // kinda fragile as they may change that output but it gives us what we need
+  // for now
```

Comment:
cc @timhaines, in case you have thoughts on the above comment.

Reply:
Tangentally related to https://github.com/percy/percy-storybook/issues/50

Reply:
I'd love to see https://github.com/storybooks/storybook/issues/4878 become a thing - then we could remove puppeteer from percy-storybook's dependencies, and it's speed and simplicity would improve a lot.

For now, I think you're okay to piggyback off what we have in place.  I'll let both of you know if/when we change it - and will make sure it's noted in the release notes.

@BPScott thanks for your notes in that issue too.